lib/api/core/plugins/workerReady.js

```diff
@@ -45,6 +67,10 @@ module.exports = function ready () {
       else if (typeof plugin[plugin.hooks[packet.data.event]] === 'function') {
         plugin[plugin.hooks[packet.data.event]](packet.data.message, packet.data.event);
       }
+      else {
+        // eslint-disable-next-line no-console
+        console.error(`Unable to register "${plugin.hooks[packet.data.event]}": not a function nor an array of functions. Ignoring...`);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 136 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)